BluetoothLeChat/app/src/main/res/layout/fragment_device_scan.xml

```diff
@@ -14,8 +14,7 @@
      See the License for the specific language governing permissions and
      limitations under the License.
 -->
-<FrameLayout
-    xmlns:android="http://schemas.android.com/apk/res/android"
+<FrameLayout xmlns:android="http://schemas.android.com/apk/res/android"
```

Comment:
If I put others all on a new line, I'm usually consistent. I would put this on a newline too to make it more readable.